projects/driver_controls/test/test_ads1015.c

```diff
@@ -1,18 +1,18 @@
 #include <stdint.h>
 #include <stdio.h>
 #include "ads1015.h"
+#include "ads1015_def.h"
+#include "delay.h"
 #include "gpio_it.h"
 #include "i2c.h"
 #include "interrupt.h"
 #include "log.h"
+#include "soft_timer.h"
 #include "unity.h"
 
-static Ads1015Storage s_storage;
-static GPIOAddress s_ready_pin = { GPIO_PORT_B, 2 };
-
 static void prv_callback(Ads1015Channel channel, void *context) {
-  Ads1015Storage *storage = context;
-  LOG_DEBUG("channel %d = channel %d\n", storage->current_channel, channel);
+  uint8_t *callback_called = context;
+  (*callback_called) = true;
```

Comment:
Shouldn't this be a bool?

You know you don't need to make everything uint8

Reply:
LOL that wasnt intentional!